tests/hermes_prolong/test_runtime_contract.py

```
@@ -0,0 +1,241 @@
+                sys.path.pop(0)
+
+        with tempfile.TemporaryDirectory() as directory:
+            home = Path(directory)
+            plugin_dir = home / "plugins" / "prolong"
+            plugin_dir.parent.mkdir(parents=True)
+            plugin_dir.symlink_to(PLUGIN_SOURCE, target_is_directory=True)
+            (home / "config.yaml").write_text(
+                "plugins:\n  enabled:\n    - prolong\n",
+                encoding="utf-8",
+            )
+            setattr(state_module, "_state_db", None)
```

Ruff (0.16.1)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)